### PR TITLE
Feature - set value of this.accepts({ func: ... }) as either a functi…

### PR DESCRIPTION
**Pull request by patowens**
Allows you to set the value of func within this.accepts({ signal: .., desc: .. , func: ... })  as either 
a) a function 
b) function name as a string

Option A:  define your function on func.

```
this.accepts({
  signal: 'do-foo',
  desc: 'does the foo',
  data: 'An object containing all the goodies needed to do the foo',
  func: () => { // ... do the foo } 
});
```

Or option B:  set the value of func to a string.  Let's you structure your code differently.  

```
this.accepts({
  signal: 'do-foo',
  desc: 'does the foo',
  data: 'An object containing all the goodies needed to do the foo',
  func: "doFoo"
});

this.doFoo = () => {
  // ..do the foo
};
```

### CODE REVIEW

File: index_src.js [hunk 1]
```diff
@@ -96,12 +96,15 @@ function RukusSignal(RukusApp, riot) {
...
         //   data : 'data accepted description',
-        //   func : function to call
+        //   func : function to call OR string name of function to call
         // }
         obj.accepts = (...signals) => {
             signals.forEach(function(sig) {
                 obj.__accepts[sig.signal] = sig;
-                obj.on(sig.signal, sig.func);
+                var func = typeof sig.func == 'function' ? sig.func : function() {
+                  obj[sig.func].apply(obj, arguments);
```
Comment (SomeoneWeird) [inline index_src.js:105]:
``` js
obj[sig.func](...arguments);
```

should work

File: index_src.js [hunk 2]
```diff
@@ -96,12 +96,15 @@ function RukusSignal(RukusApp, riot) {
...
         //   desc : 'sig description',
         //   data : 'data accepted description',
-        //   func : function to call
+        //   func : function to call OR string name of function to call
         // }
         obj.accepts = (...signals) => {
             signals.forEach(function(sig) {
                 obj.__accepts[sig.signal] = sig;
-                obj.on(sig.signal, sig.func);
+                var func = typeof sig.func == 'function' ? sig.func : function() {
```
Comment (tjstebbing) [inline index_src.js:104]: Consider naming the function, ie:  function acceptsWrapper() {  ... 
Makes it easier to locate in the debugger

**Comment by patowens** [comment]
Neat, I'll update with both suggestions

**Comment by SomeoneWeird** [comment]
On second thought, you don't need the other `function` wrapper. Just assign the function directly to `func`

**Pull request closed without merging**